src/components/graphs/linegraph/LineGraph.less

```diff
@@ -0,0 +1,5 @@
+.line-container {
+  background: #191a1a;
+  position: relative;
+  width: 100%;
+}
```

Comment:
Not a simple copy/paste of the old styling, I can appreciate the efforts taken to implement consistent changes across similar aspects of the app here!